code-studio/src/js/code-studio.js

```diff
@@ -2,7 +2,54 @@
  * @file Replacement for application.js.erb in dashboard as much as possible.
  *       Code added here will be loaded for (almost) every page in dashboard,
  *       so use sparingly!
+ *
+ * Note: This is included _after_ application.js.erb during our transition period,
+ * so when moving things preserve the include order as much as possible.
  */
 'use strict';
+/* global Sounds */
 
 require('./videos');
+
+(function () {
```

Comment:
This doesn't need to be wrapped in an IFFE anymore, right? Since browserify will do that for us.